lib/edge_impulse/Kconfig

```diff
@@ -81,6 +81,15 @@ config EI_WRAPPER_THREAD_PRIORITY
 	  that the thread will not block other operations in system for
 	  a long time.
 
+config EI_WRAPPER_PROFILING
+	bool "Run Edge Impulse library with profiling logging"
+	depends on LOG
+	depends on !TICKLESS_KERNEL
```

Comment:
why no tickless kernel?

Reply:
then `k_uptime_get()` returns 0

Reply:
it should work normally I expect - ticks are calculated in two ways in zephyr. you have ticks bookkeeping that add ticks coming from rtc irq but if for some reason you want to get time passing between ticks, tick count is anticipated from a rtc counter.

Reply:
rechecked, the problem is gone. Can be removed